CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+
+- Sarama instrumentation now adds `message_bug.destination` attribute. ([#124](https://github.com/signalfx/signalfx-go-tracing/pull/124))
```

Comment:
that is a real `message_bug` 😆 

```suggestion
- Sarama instrumentation now adds `message_bus.destination` attribute. ([#124](https://github.com/signalfx/signalfx-go-tracing/pull/124))
```

Reply:
:D  would've been nice to ship that changelog hah